pkg/cosign/kubernetes/secret.go

```diff
@@ -22,6 +22,8 @@ import (
 	"strings"
 
 	kubernetesclient "github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/client"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+
```

Comment:
very nit, maybe we can organize the import a bit

```go
import (
	"context"
	"fmt"
	"io/ioutil"
	"os"
	"strings"

	"github.com/pkg/errors"
        v1 "k8s.io/api/core/v1"
	k8serrors "k8s.io/apimachinery/pkg/api/errors"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
        kubernetesclient "github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/client"

	"github.com/sigstore/cosign/pkg/cosign"
)
```

Reply:
Thanks!